tetra/data/db_handler.py

```diff
@@ -22,7 +22,7 @@
 conf = cfg.CONF
 
 handlers = {
-    'postgres': PostgresClient()
+    'postgresql': PostgresClient()
```

Comment:
haha, :+1:

Reply:
the previous driver was called `postgres` so this just aligns the handler with the driver defined in the client and the engine defined in the conf file 😉

Reply:
:notebook: